Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -596,11 +672,25 @@ private void AcknowledgeEventId(Result result)
             }
         }
 
+        /// <summary>
+        /// Indicates if <see cref="OnLoggedIn"/> has been called since last not being logged in.
+        /// </summary>
         public bool IsUserLoggedIn
         {
             get { return userLoggedIn; }
         }
 
+        /// <summary>
+        /// Sets the current user's Social Presence.
+        /// 
+        /// TODO: This is not called by this manager. It either should be utilized, or moved to another manager, perhaps some sort of EOSPresenceManager.
+        /// </summary>
+        /// <param name="joinInfo">If provided, this string will be displayed to users in the Social Overlay as your current presence.
+        /// This can be used to indicate things like current level, looking for party status, etc.</param>
+        /// <param name="onLoggingOut">Indicates if the user is currently attempting or has already logged out.
+        /// If true, then if there's an error while trying to change your presence, the error will be ignored.
+        /// 
+        /// TODO: The error handling in both scenarios is almost identical. Perhaps an error state should be returned in the situation where this is false and an error is encountered.</param>
```

Comment:
If you choose to have line breaks within your comment tags, you should put the open and close tags on their own lines.

Reply:
That seems like a reasonable and clean guideline, will implement that.